Adobe/CFBundleVersionExtensionAttribute.xml

```diff
@@ -0,0 +1,19 @@
+<computer_extension_attribute>
+	<name>%NAME%Version</name>
+	<description />
+	<data_type>String</data_type>
+	<input_type>
+		<type>script</type>
+		<platform>Mac</platform>
+		<script>#!/bin/assh
```

Comment:
wow.. I copied them from another repo.. HOWEVER.. .jss is to be sorted.. will amend with another PR for them